recipes/sklearndf/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "sklearndf" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 368a42bb10c943b4626e1950c1b603164ffe108ed9ce3df93699f5e417514cb8
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip >=20.*
+    - python >=3.6,<3.9
+    - flit
+    - packaging >=20
+    - pandas >=0.24,<1.3
+    - gamma-pytools >=1.1,<2
+    - scikit-learn >=0.21,<0.24
```

Comment:
Are these actually needed in `host`? Normally just run should be enough

Reply:
Yes, to obtain the `__version__`-string, which build system needs to pip-install this package. Unfortunately the parent main pacakge already has those imports on the top level, and would otherwise fail.